ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -26,3 +26,28 @@ def swapdims(input, dim0, dim1):
     "uint32",
     "uint64",
 )
+
+
+def moveaxis(input, source, destination):
+    """
+    If either dimension input is a tuple, 
+    loop over both tuples and swap input tensor's axes individually in the correct order
+    """
+    if isinstance(source, tuple) or isinstance(destination, tuple):
+        assert len(source) == len(destination), "if either dimension input is a tuple, their size must match"
+        map = {}
+        for i, j in zip(source, destination):
+            if i in map: i = map[i]
+            if j in map: j = map[j]
+            input = ivy.swapaxes(input, i, j)
+            map[i] = j; map[j] = i
+        return input
+
+    return ivy.swapaxes(input, source, destination)
+
+
+movedim.unsupported_dtypes = (
```

Comment:
movedim is not defined

Reply:
my bad, that was a typo since movedim and moveaxis are similar, it's fixed now;

btw, I think you're also reviewing my other PR `movedim` which is an alias of this PR `moveaxis`; that PR is #3023

Reply:
If this is the case it may not be necessary to write code for both functions, as you can just create an alias of one another

Reply:
sure, what's the policy for creating alias?

Reply:
you can just have one as the alias of another `func_alias = func_original`